integration/tests/params.go

```diff
@@ -245,7 +247,7 @@ func (p *Params) attemptSettingUserID() {
 		return
 	}
 
-	out, err := clusterProvider.Provider.STS().GetCallerIdentity(&sts.GetCallerIdentityInput{})
+	out, err := clusterProvider.Provider.STSV2().GetCallerIdentity(context.TODO(), &sts.GetCallerIdentityInput{})
```

Comment:
TODO is a marker that we will do something with this later on. I would prefer `Background` which is mire generally used as something that doesn't define an actual context to be used later. The TODO context was defined for the reason that it shows up in TODO searches. :)

Reply:
I'd argue that `context.TODO()` is better suited here and in other places where a context is not available. 

> Code should use context.TODO when it's unclear which Context to use or it is not yet available (because the surrounding function has not yet been extended to accept a Context parameter).

Reply:
I'd argue that `context.TODO()` is better suited here and in other places where a context is not available. 

> Code should use context.TODO when it's unclear which Context to use or it is not yet available (because the surrounding function has not yet been extended to accept a Context parameter).

Reply:
Hmm. Alright, fair point.